fastchat/llm_judge/common.py

```diff
@@ -10,7 +10,8 @@
 import re
 import time
 from typing import Optional
-
+import litellm 
+from litellm import completion
```

Comment:
Import litellm under the if branch to make users without litellm still able to import and run this file.